sql/postgres/20220406.sql

```diff
@@ -0,0 +1,98 @@
+create unique index if not exists repositories_by_name on repositories(repository_name);
+
+delete from subjects where subject is null;
+drop index if exists subjects_by_subject;
+create unique index if not exists subjects_by_subject_language on subjects(subject,language);
+
+delete from tags where tag is null;
+delete from tags where tag = '';
```

Comment:
Why is tag the only field that we are concerned about the value being an empty string? And it's also the only one in this "Xs_by_X" list where we don't drop the existing index. Was this line supposed to be `drop index if exists tags_by_tag;` ? Same for sqlite.

Reply:
It's the only table that has duplicate rows for both null and '' values. Yes, the drop index was missing thanks for catching that.